### DISCUSSION TITLE
UTM fails to launch any VM on M1 Mac

### DISCUSSION THREAD

**Discussion by vedanta28** [utmapp/UTM#4237] [UTM Help (macOS)]
Now I wanted to run Ubuntu on my M1 Mac, and I followed the steps mentioned on this website <https://mac.getutm.app/gallery/ubuntu-20-04>. Now, the issue is after completing the "Creating a new virtual machine" step and trying to run the VM, I get an error.

Configuration:
* UTM Version: 3.2.4 (58)
* OS Version: macOS Monterey (Version 12.4 21F79)
* Apple Silicon

<img width="799" alt="image" src="https://user-images.githubusercontent.com/74861233/181257268-8502ed09-72aa-4f17-8540-69afde355e41.png">

This is the step where I am stuck, after configuring the VM, I cannot run it.

**Comment by MDNich** [comment]
Thats because you have internet sharing on....vm can't start a network share with the guest system if macOS Internet Sharing is on (Preferences -> Sharing -> Internet Sharing)

**Comment by vedanta28** [comment]
@MDNich I have done this it seemed to have worked, but the thing is the installation of the server stops. Also, Instead of having Install Ubuntu Server, I have Try or Install Ubuntu Server after the boot screen. I have also checked my storage, it seems to be 15 GBs off. I have deleted all the previous VM instances, from UTM menu, but I think a faulty installation still remains. Do you know were are the Ubuntu Installation stored? So, I can clear that up.

**Comment by MDNich** [comment]
@vedanta28 UTM VMs are located in ~/Library/Containers/com.utmapp.UTM/Data/Documents/
Check there (either cd in terminal or Shift Cmd G in Finder, though seeing that you are installing Ubuntu Server I would imagine you know how to find a folder on your Mac :)
